meetings/2018-01-04.md

```diff
@@ -0,0 +1,54 @@
+# Node.js Foundation Security WorkGroup Meeting 2018-01-04
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=PGQio_Zhhiw
+* **GitHub Issue**: https://github.com/nodejs/security-wg/issues/84
+* **Minutes Google Doc**: https://docs.google.com/document/d/1GxwiNP2f7bTpVfeQ9sBD_WrN5drWNs_a4USrmimiiC0
+
+## Present
+
+* Michael Dawson (@mhdawson)
+* Colin Ihrig (@cjihrig)
+* Bryce Baril (@brycebaril)
+* Vladimir de Turckheim (@vdeturckheim)
+* Michael Alexander (@mgalexander)
+* Liran Tal (@lirantal)
```

Comment:
I think it misses @grnd

Reply:
@vdeturckheim right. i joined 10 mins in.